src/main/java/step02/code/domain/Lotto.java

```diff
@@ -65,6 +65,10 @@ public int match(Lotto winningNumber) {
     return LOTTO_SIZE - lotto.size();
   }
 
+  public boolean isMatchBonus(int bonus) {
+    return lotto.stream().anyMatch(number -> number.isMatch(bonus));
```

Comment:
```suggestion
    return lotto.contains(bonus);
```
가끔은 더 같단한 방법이 있기도 하죠 😀